torch/_tensor.py

```diff
@@ -1158,7 +1299,8 @@ def __len__(self):
             )
         return self.shape[0]
 
-    def __iter__(self):
+    # TODO: This raises an enormous amount of errors, need to fix as separate PR
```

Comment:
Need to dig into this more carefully, unclear why it's raising such a large amount of errors (are they false positives, or revealing something deeper) ...